tests/testthat/tests.R

```diff
@@ -1,6 +1,6 @@
 library(EpiEstim)
 library(testthat)
-library(compare)
+library(compare) # TODO: replace this with testthat::expect_equal_to_reference()
```

Comment:
We should either do this, or add an issue that it needs doing, otherwise it'll be forgotten about

Reply:
Issue created: https://github.com/annecori/EpiEstim/issues/72